tests/src/Functional/NewsIntegrationTest.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_starter_content\Functional;
+
+use Drupal\Tests\BrowserTestBase;
+use Drupal\Tests\media\Traits\MediaTypeCreationTrait;
+use Drupal\media\Entity\Media;
+use Drupal\file\Entity\File;
+use Drupal\Tests\TestFileCreationTrait;
+
+/**
+ * Ensure OE Starter Content News are created.
+ *
+ * @group oe_starter_content
+ */
+class NewsIntegrationTest extends BrowserTestBase {
+
+  use MediaTypeCreationTrait;
+  use TestFileCreationTrait {
+    getTestFiles as drupalGetTestFiles;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  protected static $modules = [
+    'datetime',
+    'media',
+    'system',
+    'user',
+    'node',
+    'oe_starter_content_news',
+    'config',
+    'entity_browser',
```

Comment:
If we need to specify more than oe_starter_content_news, then we are doing something wrong in listing dependencies in the module itself.

Reply:
I've added the dependencies to the appropriate modules but we still need the base entity node for this test.